docs/release-promotion/common/uncategorized.md

```diff
@@ -0,0 +1,14 @@
+# A word on selecting partials in Ship-it
```

Comment:
Doesn't RelMan do this, not us?

Reply:
Yes, it does. It's more a reminder from us as during `60.0b1 Devedition` we were asked about this and hesitated. Then we ended up serving complete mars for a large population. I thought it's a good informative reminder if we ever get asked again. 

We can: 
a) add a comment stating this is purely for information and is done by RelMan
b) move this section somewhere else in the repo where it's less confusing.

What do you think?

Reply:
I have no strong feelings